packages/material-ui-styles/src/makeStyles/makeStyles.d.ts

```diff
@@ -2,22 +2,15 @@ import { ClassNameMap, Styles, WithStylesOptions } from '@material-ui/styles/wit
 import { Omit } from '@material-ui/types';
 import { DefaultTheme } from '../defaultTheme';
 
-/**
- * `makeStyles` where the passed `styles` do not depend on props
- */
-export default function makeStyles<Theme = DefaultTheme, ClassKey extends string = string>(
-  style: Styles<Theme, {}, ClassKey>,
-  options?: Omit<WithStylesOptions<Theme>, 'withTheme'>
-): (props?: any) => ClassNameMap<ClassKey>;
-
-/**
- * `makeStyles` where the passed `styles` do depend on props
- */
 export default function makeStyles<
   Theme = DefaultTheme,
-  Props extends {} = {},
+  Props extends object = {},
```

Comment:
Little correction while we're at it. The types we pass this to expect `object` as well.